test/parallel/test-fs-promises-file-handle-readFile.js

```diff
@@ -10,7 +10,7 @@ const {
   open,
   readFile,
   writeFile,
-  truncate
+  truncate,
```

Comment:
```suggestion
  truncate
```
Unrelated change

Reply:
Personally I'm in favour of not spending mental energy on stylistic changes that are not enforced as part of linting rules.